mlibrary/get-this
# Layout cleanup

# Overview
The website build had a short deadline, leaving very little room for QA. This pull request goes over the front-end portion of the site with a fine-tooth comb, cleaning up and reorganizing code, while also resolving most accessibility issues.

## Anything Else?
### Footer
A footer has been added to match what is seen in Catalog Browse, and Search (in a sense).

### Chat
To also match what is seen in the above sites.

## Testing
- Run the tests to make sure they pass (`docker-compose run --rm web bundle exec rspec`).
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Build the CSS (`docker-compose run --rm web npm run build`)
  - Look through the site and see if anything looks borked.
